api/types/events/events_test.go

```diff
@@ -99,6 +102,28 @@ func TestTrimToMaxSize(t *testing.T) {
 				DatabaseQuery: `{"a": "b","a":`,
 			},
 		},
+		// UserLogin
+		{
+			name:    "UserLogin event with error",
+			maxSize: 3000,
+			in: &UserLogin{
+				Status: Status{
+					Error:       strings.Repeat("A", 2000),
+					UserMessage: strings.Repeat("A", 2000),
+				},
+			},
+			want: &UserLogin{
+				Status: Status{
+					Error:       strings.Repeat("A", 1336),
+					UserMessage: strings.Repeat("A", 1336),
```

Comment:
Should we add some indication that the field was trimmed? Maybe "AAAA..." or some other marker?

Reply:
I thought about that when adding the last message trimmer but decided to keep them consistent. Rather than introduce that change here I'd like to introduce that change in a follow up PR that includes other refactoring.

Reply:
Ack.